claim/claim_test.go

```diff
@@ -45,18 +45,23 @@ func TestUpdate(t *testing.T) {
 }
 
 func TestValidName(t *testing.T) {
-	is := assert.New(t)
-
 	for name, expect := range map[string]bool{
 		"M4cb3th":               true,
-		"Lady MacBeth":          false, //spaces illegal
+		"Lady MacBeth":          false, // spaces illegal
 		"3_Witches":             true,
 		"Banquø":                false, // We could probably loosen this one up
```

Comment:
Unrelated to this PR, but `ø` is part of L and here it isn't approved. Group S is spacing and is also approved. In fact, the only group that seems to not be allowed here is `C` so none of the tests here should fail. Am I missing something?

Reply:
On that note, my understanding is that, if you are using `regex` (and thus `re2`) you can get away using unicode character classes explicitly and probably get away with `[\pL\pM\pN\pP\pS\pZ]*` or so...

Reply:
I'll open a new issue for this, thanks!

Reply:
See #126